Makefile

```diff
@@ -1,7 +1,7 @@
 DCURL_DIR := third_party/dcurl
 DCURL_LIB := $(DCURL_DIR)/build/libdcurl.so
 MOSQITTO_DIR := third_party/mosquitto
-MOSQITTO_LIB := $(MOSQITTO_DIR)/lib/libmosquitto.so.1
+MOSQITTO_LIB := $(MOSQITTO_DIR)/lib/libmosquitto.so.*
```

Comment:
Why?

Reply:
[rpath issue](https://wiki.debian.org/RpathIssue) is really critical for dynamic linking, and you must explicitly specify the version number.